packages/next/build/index.ts

```diff
@@ -197,6 +197,7 @@ export default async function build(
           pagesDir,
           !ignoreTypeScriptErrors,
           !config.images.disableStaticImages,
```

Comment:
I think you can omit this `!config.images.disableStaticImages` parameter and perform the logic inside the function since you'll have the complete config